R/Reactant/build_tarballs.jl

```diff
@@ -312,7 +315,7 @@ augment_platform_block="""
     """
 
 # for gpu in ("none", "cuda", "rocm"), mode in ("opt", "dbg"), platform in platforms
-for gpu in ("none", "cuda"), mode in ("opt", "dbg"), cuda_version in ("none", "12.1", "12.6"), platform in platforms
+for gpu in ("none", "cuda"), mode in ("opt", "dbg"), cuda_version in ("none", "12.1", "12.3", "12.6"), platform in platforms
```

Comment:
Is this what you suggested in https://github.com/LuxDL/Lux.jl/issues/1225#issuecomment-2606017095?

Reply:
mostly, but I think we can drop 12.1 even